tasks/engine_setup.yml

```diff
@@ -0,0 +1,65 @@
+---
+- block:
+  - name: Set answer file path
+    set_fact:
+      answer_file_path: "/tmp/answerfile-{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}.txt"
```

Comment:
This is insecure. Please use mktemp or ansible tempfile module. If you require root, you can also use /root/something .

Reply:
Why is this insecure?